src/app.js

```diff
@@ -6,10 +6,13 @@ const hd = new Holidays();
 
 hd.init('DK');
 
-app.use(express.static('assets'));
+app.use(express.static('src/assets'));
 
+app.set('views', 'src/views');
 app.set('view engine', 'pug');
 
+const port = process.env.PORT || 8080;
```

Comment:
what is dis process.env.PORT?

Reply:
Det er en config fil som Node bruger [Docs here](https://nodejs.org/dist/latest-v8.x/docs/api/process.html#process_process_env), som gør at vi kan ændre porten fra Heroku's web interface hvis det bliver nødvendigt :)